src/wallet/rpcwallet.cpp

```diff
@@ -4489,7 +4489,20 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "    }, ... ]\n"
             "3. minconf               (numeric, optional, default=1) Only use funds confirmed at least this many times.\n"
             "4. fee                   (numeric, optional, default=" + strprintf("%s", FormatMoney(DEFAULT_FEE)) + ") The fee amount to attach to this transaction.\n"
-            "5. allowRevealedAmounts  (bool, optional, default=false) Permit cross-shielded-pool transfers, which will publicly reveal the amount(s) crossing pool boundaries.\n"
+            "5. privacyPolicy         (string, optional, default=\"LegacyCompat\") Policy for what information leakage is acceptable.\n"
+            "                         One of the following strings:\n"
+            "                               - \"FullPrivacy\": Only allow fully-shielded transactions (involving a single shielded pool).\n"
+            "                               - \"LegacyCompat\": If the transaction involves any UAs, this is equivalent to \"FullPrivacy\".\n"
+            "                                 Otherwise, this is equivalent to \"AllowFullyTransparent\".\n"
+            "                               - \"AllowRevealedAmounts\": Allow funds to cross between shielded pools.\n"
+            "                               - \"AllowRevealedRecipients\": Allow transparent recipients. Implies \"AllowRevealedAmounts\".\n"
+            "                               - \"AllowRevealedSenders\": Allow transparent UTXOs to be spent. Implies \"AllowRevealedAmounts\".\n"
+            "                               - \"AllowFullyTransparent\": Allow transaction to have both transparent UTXOs and transparent recipients.\n"
+            "                                 Implies \"AllowRevealedSenders\" and \"AllowRevealedRecipients\".\n"
+            "                               - \"AllowLinkingAccountAddresses\": Allow selecting transparent coins from the full account,\n"
+            "                                 rather than just the transparent funds sent to the provided UA. Implies \"AllowRevealedSenders\".\n"
```

Comment:
```suggestion
            "                                 rather than just the funds sent to the transparent address in the provided Unified Address. Implies \"AllowRevealedSenders\".\n"
```